masterdata_context.md

```diff
@@ -365,6 +366,210 @@ data | hash |
 }
 ```
 
+### Masterdata.TenancyAgreement.Create
+
+This message is sent from an external message publisher to a GARAIO REM instance and allows to create a tenancy agreement. Set the recipient property in the headers, eg "grem_wincasa". Alwaays required attributes are noted in the remarks. Depending on the tenancy agreement template configuration, additional attributes may be required.
+
+GARAIO REM replies with a standard Accepted / Rejected message containing the tenancyAgreementReference of the created tenancy agreement or reject reasons
```

Comment:
Not important but I'd recommend in putting API terms or command in backticks for better readability.

```suggestion
GARAIO REM replies with a standard Accepted / Rejected message containing the `tenancyAgreementReference` of the created tenancy agreement or reject reasons
```